app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/Model/DataSetInterface.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.HumanLearning2021.HumanLearningApp.Model
+
+import android.graphics.drawable.Drawable
+
+
+/**
+ * An interface representing the part of the model interacting with data sets
+ */
+interface DataSetInterface {
+    suspend fun getPicture(category: Category): CategorizedPicture
```

Comment:
```suggestion
    suspend fun getPicture(category: Category): CategorizedPicture?
```
I realized when experimenting with my implementation: if no picture is available, it might make sense to return null instead of throwing an exception. Compare this to the plural case: if this was "getPictures", we could return an empty list.